index.js

```diff
@@ -11,241 +11,352 @@ inputarea.sortable({
+          formatsCommand.push('46')
+          formatsExample.push('color-bg-cyan')
           break
         case 'Light gray':
-          formats.push('47')
+          formatsCommand.push('47')
+          formatsExample.push('color-bg-light-gray')
           break
         case 'Dark gray':
-          formats.push('100')
+          formatsCommand.push('100')
+          formatsExample.push('color-bg-dark-gray')
           break
         case 'Light red':
-          formats.push('101')
+          formatsCommand.push('101')
+          formatsExample.push('color-bg-light-red')
           break
         case 'Light green':
-          formats.push('102')
+          formatsCommand.push('102')
+          formatsExample.push('color-bg-light-green')
           break
         case 'Light yellow':
-          formats.push('103')
+          formatsCommand.push('103')
+          formatsExample.push('color-bg-light-yellow')
           break
         case 'Light blue':
-          formats.push('104')
+          formatsCommand.push('104')
+          formatsExample.push('color-bg-light-blue')
           break
         case 'Light magenta':
-          formats.push('105')
+          formatsCommand.push('105')
+          formatsExample.push('color-bg-light-magenta')
           break
         case 'Light cyan':
-          formats.push('106')
+          formatsCommand.push('106')
+          formatsExample.push('color-bg-light-cyan')
           break
         case 'White':
-          formats.push('107')
+          formatsCommand.push('107')
+          formatsExample.push('color-bg-white')
           break
         default:
           command += '\\[\\e[48;5;' + $(this).attr('data-bg-color') + 'm\\]'
+          // Use given color
+          colorFG = $(this).attr('data-bg-color')
       }
+    } else {
+      formatsExample.push('color-bg-black')
     }
 
-    if (formats.length > 0) {
-      command += '\\[\\e[' + formats.join(';') + 'm\\]'
-
-      if (formats.length === 1 && formats[0] === '0') {
+    if (formatsCommand.length > 0) {
+      command += '\\[\\e[' + formatsCommand.join(';') + 'm\\]'
+      if (formatsExample.indexOf('color-'))
+      colorFG = (colorFG === null ? '' : `style="color:${parseColor(colorFG)};"`)
+      colorBG = (colorBG === null ? '' : `style="color:${parseColor(colorBG)};"`)
+      example += `<span class="${formatsExample.join(' ')}"${colorFG}${colorBG}>`
+      if (formatsCommand.length === 1 && formatsCommand[0] === '0') {
         requireReset = false
       } else {
         requireReset = true
       }
     } else {
       requireReset = false
+      example += '<span>'
     }
 
     switch ($(this).html().replace(/\s*<i.*<\/i>/gi, '').trim()) {
       case 'Username':
         command += '\\u'
+        example += 'Scriptim'
         break
       case 'Hostname (short)':
         command += '\\h'
+        example += 'Scriptim-PC'
         break
       case 'Hostname':
         command += '\\H'
+        example += 'Scriptim-PC'
```

Comment:
These examples are poorly chosen. I think you should simply use `user` and `hostname`. Especially since a hostname usually does neither contain capital letters nor non-alphanumeric characters (see [RFC 1178](https://tools.ietf.org/html/rfc1178)).